DevelopGitHub/GitHubExtension/GitHubExtension.Activity.External.WebAPI/Controllers/ActivityController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Security.Claims;
+using System.Threading.Tasks;
+using System.Web.Http;
+using GitHubExtension.Activity.External.WebAPI.Services;
+using GitHubExtension.Security.WebApi.Library.Exceptions;
+using Microsoft.AspNet.Identity;
+using Newtonsoft.Json.Linq;
+
+namespace GitHubExtension.Activity.External.WebAPI.Controllers
+{
+    [Authorize]
+    public class ActivityController : ApiController
+    {
+        public const string GetGitHubActivityRoute = "api/activity/external";
```

Comment:
Extract to class which will be hold all route constant related to bounded context.